src/main/java/no/unit/nva/institution/proxy/HttpExecutorImpl.java

```diff
@@ -34,13 +37,14 @@
 
     public static final String NVA_INSTITUTIONS_LIST_CRAWLER = "NVA Institutions List Crawler";
     public static final String INSTITUTIONS_URI_TEMPLATE =
-        "https://api.cristin.no/v2/institutions?country=NO" + "&per_page=1000000&lang=%s";
+        "https://api.cristin.no/v2/institutions?country=NO" + "&per_page=1000000&lang=%s&cristin_institution=true";
```

Comment:
This is the updated URL.

Reply:
Remember to remove the filter InstitutionUtils 39

Reply:
We can dispense with the three of the zeros as the API only returns 1000.